drools-yaml-rules-core/src/main/java/org/drools/yaml/core/jpy/JpyAstRulesEngine.java

```diff
@@ -0,0 +1,68 @@
+package org.drools.yaml.core.jpy;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.drools.yaml.core.RulesExecutor;
+import org.drools.yaml.core.RulesExecutorContainer;
+import org.kie.api.runtime.rule.Match;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import static org.drools.yaml.core.RulesExecutor.OBJECT_MAPPER;
+
+public class JpyAstRulesEngine {
+
+    private Iterator<Map<String, Map>> lastResponse = Collections.emptyIterator();
+
+    public long createRuleset(String ruleSetName, String rulesetString) {
```

Comment:
ruleSetName is unused, how do we pass it in @mariofusco ? is this supported or do we need to make a small change?

Reply:
We have use cases with multiple rulesets so it might make sense to store the name in the session information it will aid in debugging. Or we could just pass the entire ruleset with the name and collection of rules as json and you can parse it on your side.